package.json

```
@@ -38,7 +38,7 @@
     "package.json": "sort-package-json"
   },
   "dependencies": {
-    "@defi-wonderland/prophet-core": "0.0.0-0954a47f",
+    "@defi-wonderland/prophet-core": "github:defi-wonderland/prophet-core#de18a05",
```

[Latest canary](https://www.npmjs.com/package/@defi-wonderland/prophet-core?activeTab=versions)
```suggestion
    "@defi-wonderland/prophet-core": "0.0.0-4d28a0ec",
```